crates/ide_db/src/tests/sourcegen_lints.rs

```diff
@@ -21,6 +21,7 @@ fn sourcegen_lint_completions() {
 pub struct Lint {
     pub label: &'static str,
     pub description: &'static str,
+    pub children: &'static [&'static str],
```

Comment:
This could be `Option<&'static [&'static str]>,` but there is no real difference